src/main/java/com/deavensoft/paketomat/user/UserServiceImpl.java

```diff
@@ -11,9 +16,11 @@
 @Service
 public class UserServiceImpl implements UserService {
     private UserRepository userRepository;
+    private CenterService centerService;
     @Autowired
-    public UserServiceImpl(@Qualifier("user") UserRepository userRepository){
+    public UserServiceImpl(@Qualifier("user") UserRepository userRepository,CenterService centerService){
         this.userRepository = userRepository;
+        this.centerService=centerService;
     }
```

Comment:
- [ ] Remove it

Reply:
Done.

Reply:
Please delete the annotation @Autowired.